packages/ffe-searchable-dropdown-react/src/MultiSelect/reducer.ts

```diff
@@ -0,0 +1,187 @@
+                    };
+                }
+                return state;
+            }
+            case 'InputChange': {
+                const { noMatch, listToRender } = getListToRender({
+                    inputValue: action.payload?.inputValue ?? '',
+                    searchAttributes,
+                    maxRenderedDropdownElements,
+                    dropdownList,
+                    noMatchDropdownList,
+                    searchMatcher,
+                    showAllItemsInDropdown: false,
+                });
+
+                return {
+                    ...state,
+                    isExpanded: true,
+                    inputValue: action.payload?.inputValue ?? '',
+                    listToRender,
+                    highlightedIndex:
+                        action.payload?.inputValue?.trim() === '' ||
+                        listToRender.length === 0
+                            ? -1
+                            : 0,
+                    noMatch,
+                };
+            }
+            case 'ToggleButtonPressed':
+                return {
+                    ...state,
+                    isExpanded: !state.isExpanded,
+                };
+            case 'ItemOnClick':
+            case 'InputKeyDownEnter':
+                if (action.payload?.item) {
+                    const { noMatch, listToRender } = getListToRender({
+                        inputValue: '',
+                        searchAttributes,
+                        maxRenderedDropdownElements,
+                        dropdownList,
+                        noMatchDropdownList,
+                        searchMatcher,
+                        showAllItemsInDropdown: true,
+                    });
+                    return {
+                        ...state,
+                        isExpanded: true,
+                        highlightedIndex:
+                            state.inputValue.trim() === ''
+                                ? (action.payload?.highlightedIndex ?? -1)
+                                : -1,
+                        selectedItems: getNewList(
+                            state.selectedItems,
+                            action.payload.item,
+                            action.payload?.actionType ?? 'selected',
+                        ),
+                        listToRender: listToRender,
+                        inputValue: '',
+                        noMatch,
+                    };
+                }
+                return state;
+            case 'InputKeyDownArrowDown':
+            case 'InputKeyDownArrowUp': {
+                const focusedElement = document.activeElement;
+                if (focusedElement?.getAttribute('role') === 'combobox') {
+                    return {
+                        ...state,
+                        isExpanded: true,
+                        highlightedIndex:
+                            action.payload?.highlightedIndex ?? -1,
+                    };
+                }
```

Comment:
Dette er workaround for å sikre at man ikke har chip valgt samtidig som man kan gå nedover i lista. Håper ikke det brekker noe annet.

Reply:
Det er akkurat slika ting det er veldig fint att ha tester på :). Derfor jag er slitsom med det. Brekker man noe når man endrer. Det er greit att veta.